release-notes/2024-01/README.md

```diff
@@ -0,0 +1,187 @@
+# SystemLink Enterprise release 2024-01 Release Notes
+
+The 2024-01 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Added support for SystemLink Client 2024 Q1.
+- You can @ mention other users in comments on test results and data spaces.
+- You can trigger test data extraction routines from the Product details page.
+- You can trend step data from the Results details page directly into a data space.
+- You can monitor disk utilization for your JupyterHub user data in the main status bar at the bottom of the window.
+- You can add keywords and properties in test result steps.
+
+## Helm Chart Breaking Changes
+
+- `systemlink 0.21.4`
+    - The license service and the license UI have been removed from the product.
+    - Settings related to `license:` in Helm values files are no longer required and can be removed from existing configurations.
+    - Administrators should delete persistent volume claims `datadir-{release}-license-mongodb-0`,  `datadir-{release}-license-mongodb-1`, and `datadir-{release}-license-mongodb-2` if they exist.
+
+- `TestMonitorService 0.18.0`
```

Comment:
@Christian-Nunnally can you check this part of the release notes and make sure it's accurate?I had to rewrite some things, and I want to make sure that its not an incorrect statement.

Reply:
@prestwick This isn't accurate. Migrations are performed by the k8s job regardless of credentials, and in other public facing docs we have been saying table owner rather than ALTER_TABLE permissions. The text here should be close to what we put for the last migration we did [for 2023-11](https://github.com/ni/install-systemlink-enterprise/tree/main/release-notes/2023-11). The [ni.com documentation](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/config-systemlink-enterprise.html#GUID-22D7F822-FF82-436A-9458-DA1D33334886__GUID-A2CA66A5-7E59-419B-8638-48E7E7A3963A) was also updated after that previous version to include some of the same information in case you'd rather link to that.

Reply:
Its also two columns, one for keywords and one for properties

Reply:
Thanks for the feedback. I'll reuse the text from the 2023-11 release and reference ni.com